src/Atmos/Model/AtmosModel.jl

```diff
@@ -47,7 +48,8 @@ A `BalanceLaw` for atmosphere modeling.
                boundarycondition, init_state)
 
 """
-struct AtmosModel{FT,O,RS,T,M,P,R,S,BC,IS,DC} <: BalanceLaw
+struct AtmosModel{FT,PS,O,RS,T,M,P,R,S,BC,IS,DC} <: BalanceLaw
```

Comment:
Use a more distinctive variable name than ps/PS for parameter set throughout. Otherwise this invites confusion with surface pressure.

Reply:
Is `param_set` okay?